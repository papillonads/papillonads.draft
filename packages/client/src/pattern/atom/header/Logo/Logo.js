import React from 'react'
import { shape, string } from 'prop-types'
import cn from 'classnames'
import styles from './logo.scss'

const Logo = ({ className }) => {
  const logoClassList = cn(className, styles.logo)

  return <div className={logoClassList}>Logo</div>
}

Logo.defaultProps = {
  className: '',
}

Logo.propTypes = shape({
  className: string,
}).isRequired

export default Logo
