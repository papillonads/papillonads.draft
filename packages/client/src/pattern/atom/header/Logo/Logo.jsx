import React from 'react'
import { string } from 'prop-types'
import cn from 'classnames'
import SvgIconGroup from '../Icon/SvgIconGroup'

const Logo = ({ className }) => {
  const logoClassList = cn(className)

  return (
    <div className={logoClassList}>
      <SvgIconGroup
        name="logo"
        width="143"
        height="26"
        viewBox="0 0 528 96"
        pathProps={[{ fill: '#2D3C4D' }, { fill: '#EEA566' }, { fill: '#2D3C4D' }]}
      />
    </div>
  )
}

Logo.defaultProps = {
  className: '',
}

Logo.propTypes = {
  className: string,
}.isRequired

export default Logo
