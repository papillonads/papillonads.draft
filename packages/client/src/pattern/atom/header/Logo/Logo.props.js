import { string } from 'prop-types'

export const logoDefaultProps = {
  className: '',
}

export const logoPropTypes = {
  className: string,
}
