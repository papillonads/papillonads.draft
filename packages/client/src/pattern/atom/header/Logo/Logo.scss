.logo {
  padding: 4px 12px 4px 4px;
}
