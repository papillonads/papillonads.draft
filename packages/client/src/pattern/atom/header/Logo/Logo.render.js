import React from 'react'
import Logo from './Logo'

const defaultRender = <Logo />

export { defaultRender }
