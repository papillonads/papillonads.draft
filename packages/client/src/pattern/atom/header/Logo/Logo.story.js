import { storiesOf } from '@storybook/react'
import { defaultRender } from './Logo.render'
import { getInfo } from '../../../../../../../.storybook/utility'

const defaultRenderInfo = getInfo('default')

storiesOf('Patttern/Atom/Header/Logo', module).add(defaultRenderInfo.text, () => defaultRender, defaultRenderInfo.parameters)
