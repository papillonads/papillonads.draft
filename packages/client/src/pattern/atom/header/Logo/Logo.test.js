import { defaultRender } from './Logo.render'

describe('<Logo />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
