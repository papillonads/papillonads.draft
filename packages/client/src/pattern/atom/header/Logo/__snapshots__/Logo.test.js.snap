// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <svg
    className=""
    height="26"
    viewBox="0 0 528 96"
    width="143"
  >
    <g
      fillRule="evenodd"
    >
      <path
        d="M119.692 23.506v6.355l3.857.219-2.328 33.089c-.219 1.826-.583 2.337-1.965 2.556l-2.256.22v6.135h17.319v-6.355l-3.639-.219 1.456-26.296h.145l10.26 27.684h7.641l10.552-28.049h.218l1.237 24.397c.073 1.826-.218 2.264-1.601 2.483l-2.256.22v6.135h17.392v-6.355l-3.711-.219-2.256-33.235c-.219-1.534.51-2.191 1.892-2.41l2.183-.22v-6.135H157.75l-11.133 30.897h-.219l-10.697-30.897h-16.009zm92.488 13.221c-5.53-2.118-10.842-2.63-13.389-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.553 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zM202.14 59.955c0 2.922-2.11 5.624-6.113 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.567.365 4.513.73v17.604zm37.111-17.677c.8 0 1.383.073 1.82.22.509.219.654.584.654 1.752v4.821l8.223-.584V35.339c-1.965-.876-4.584-1.242-6.695-1.242-6.04 0-9.168 3.653-10.333 6.574h-.145v-5.624h-13.317v6.136l2.474.219c1.237.073 1.674.584 1.674 2.045V63.68c0 1.68-.218 2.191-1.746 2.41l-2.329.22v5.77h20.157v-6.063l-6.04-.292V49.071c0-3.871 2.11-6.793 5.603-6.793zm28.307 14.39c1.528-.22 2.91-.438 4.366-1.023l6.986 16.435h12.225v-6.063l-3.93-.219-6.622-14.316c3.493-2.849 5.458-6.501 5.967-10.007l3.275-.438v-5.99h-16.082v5.916l1.892.22c.946.219 1.528.438 1.528 1.387 0 3.36-5.166 6.501-9.605 7.086V20h-14.263v6.136l2.475.219c1.382.073 1.673.584 1.673 2.118v35.353c0 1.68-.436 2.045-1.673 2.264l-2.33.22v5.77h17.392v-6.063l-3.274-.219v-9.13zm25.032-14.536h4.948v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.277-.95 9.315-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.876 1.242-2.91 0-4.438-1.169-4.438-5.478V42.132h8.659l.364-7.085h-9.023v-8.4h-5.604c-1.528 4.383-3.492 8.254-8.732 9.203l-.873 6.282zM325.771 78.8c0 1.826-.364 2.118-1.746 2.337l-2.256.293v5.77h20.375v-6.063l-6.258-.365v-9.495c1.965 1.095 4.657 1.826 7.277 1.826 10.26 0 15.572-7.89 15.572-20.672 0-11.76-4.584-18.334-13.098-18.334-4.075 0-8.15 1.461-10.406 5.479h-.145v-4.529h-13.463v6.136l2.402.146c1.164.073 1.746.73 1.746 2.118V78.8zm10.188-31.263c0-2.337 1.892-5.697 6.112-5.697 5.312 0 6.404 5.843 6.404 11.833 0 6.647-1.674 11.833-7.131 11.833-1.82 0-3.93-.657-5.385-1.753V47.537zM376.854 20h-14.262v6.136l2.401.219c1.383.073 1.747.584 1.747 2.118V63.68c0 1.826-.291 2.118-1.674 2.337l-2.329.293v5.77h17.901v-6.063l-3.784-.219V20zm40.532 16.727c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm50.792-23.228c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm16.227-17.823h4.949v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.276-.95 9.314-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.875 1.242-2.91 0-4.439-1.169-4.439-5.478V42.132h8.66l.363-7.085h-9.023v-8.4h-5.603c-1.528 4.383-3.493 8.254-8.732 9.203l-.874 6.282zm49.192-.657c.873.292 1.018.803 1.018 1.972v2.995l7.204-.585v-9.13c-3.201-1.753-7.567-2.63-11.57-2.63-8.732 0-15.208 4.164-15.208 11.98 0 7.888 5.53 9.568 10.77 10.956 4.074.95 6.912 1.68 6.912 4.82 0 3.142-2.255 4.237-5.675 4.237-1.456 0-2.984-.292-3.93-.657-.873-.292-1.091-.73-1.091-1.826V60.32l-7.277.584v9.715c2.838 1.534 8.44 2.484 12.88 2.484 9.387 0 15.354-4.456 15.354-12.345 0-7.596-5.749-9.276-11.061-10.883-3.857-1.169-6.986-1.753-6.986-5.04 0-2.557 2.183-3.798 5.094-3.798 1.164 0 2.474.073 3.566.438z"
        fill="#2D3C4D"
      />
      <path
        d="M48.018 0C21.498 0 0 21.492 0 48s21.497 48 48.018 48 48.019-21.492 48.019-48S74.539 0 48.018 0"
        fill="#EEA566"
      />
      <path
        d="M64.19 36.65a8.877 8.877 0 0 1 0 12.782L51.196 62.09c-3.623 3.533-9.497 3.533-13.12 0l19.554-19.05L44.64 30.38a8.874 8.874 0 0 1 0-12.78l19.55 19.05zM57.21 77.12H29.771c0-4.995 4.12-9.047 9.206-9.047H66.417c0 4.996-4.121 9.047-9.206 9.047"
        fill="#2D3C4D"
      />
    </g>
  </svg>
</div>
`;
